lib/Button/Button.js

```diff
@@ -31,57 +31,60 @@ export const propTypes = {
   type: PropTypes.string,
 };
 
-const defaultProps = {
-  buttonStyle: 'default',
-  type: 'button',
-};
-
-const Button = React.forwardRef((props, ref) => {
+const Button = React.forwardRef((
+  {
+    align,
+    allowAnchorClick,
+    autoFocus,
+    bottomMargin0,
```

Comment:
```suggestion
    allowAnchorClick,
    bottomMargin0,
```


Since `autoFocus` is being destructured out here, it isn't applied to the button when all of the `inputCustom` props are gathered up down in line 80.

Reply:
🤦